ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -10991,6 +10994,8 @@ ABoundActivity * HqlCppTranslator::doBuildActivityOutput(BuildCtx & ctx, IHqlExp
     Owned<ActivityInstance> instance = new ActivityInstance(*this, ctx, kind, expr, activityArgName);
     //Output to a variable filename is either a user result, or a computed workflow spill, both need evaluating.
 
+    instance->setUseGenericReadWrite(useGenericReadWrites);
```

Comment:
I am not quite sure what this is here for, or why this needs to be tracked in the instance.  It is currently unused.

Reply:
I am still learning how this framework operates and believed that I needed to propagate that boolean. I will remove it.